app/views/efficacy/EfficacyView.vue

```diff
@@ -0,0 +1,974 @@
+  .section-commitment {
+    background: linear-gradient(125.33deg, #355EA0 0%, #355EA0 27.36%, #5DB9AC 115.76%, #38F1E0 136.82%);
+
+    @media (max-width: $screen-md-min) {
+      margin-top: 50px;
+    }
+
+    .container {
+      padding: 60px 0;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+
+      @media (max-width: $screen-md-min) {
+        margin-left: 50px;
+        margin-right:50px;
+        max-width: calc(100% - 100px);
+      }
+
+      h2 {
+        font-family: 'Work Sans';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 46px;
+        line-height: 122%;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: $moon;
+        margin: 0 auto 50px;
+      }
+
+      p {
+        font-family: 'Work Sans';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 24px;
+        line-height: 125%;
+
+        display: flex;
+        align-items: center;
+        text-align: center;
+        letter-spacing: 0.444444px;
+        color: white;
+        margin: 0 auto 50px auto;
+
+        &.question {
+          font-weight: 700;
+          font-size: 24px;
+          display: flex;
+          align-items: center;
+          letter-spacing: 0.444444px;
+          color: $moon;
+          margin: 0 auto 30px auto;
+        }
+      }
+
+      a {
+        font-family: 'Work Sans';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 20px;
+        line-height: 23px;
+        text-align: center;
+        color: black;
+        background: $moon;
+        padding: 18px 80px;
+      }
+    }
+  }
+
+  .teachers-love .container {
+    max-width: 1260px;
+    margin: 150px auto;
+    padding: 70px 50px;
+    border-image: linear-gradient(
+            to right,
+            #3272DB,
+            #33ECC9
+    ) 1;
+    border-width: 10px;
+    border-style: solid;
+    text-align: center;
+
+    @media (max-width: $screen-md-min) {
+      margin-right: 50px;
+      margin-left: 50px;
+      max-width: calc(100% - 100px);
+    }
+
+    h2 {
+      font-family: 'Arvo';
+      font-style: normal;
+      font-weight: 700;
+      font-size: 46px;
+      line-height: 62px;
+      text-align: center;
+      letter-spacing: 2px;
+      color: $teal-light-1;
+    }
+
+    p {
+      font-family: 'Open Sans';
+      font-style: normal;
+      font-weight: 400;
+      font-size: 18px;
+      line-height: 26px;
+      text-align: center;
+      letter-spacing: 0.75px;
+      color: $teal-dark;
+    }
+
+    a {
+      font-family: 'Work Sans';
+      font-style: normal;
+      font-weight: 600;
+      font-size: 20px;
+      line-height: 23px;
+      text-align: center;
+      color: #000000;
+      padding: 18px 80px;
+      background: $dusk;
+    }
+
+    .teachers-love-boxes {
+      margin-bottom: 50px;
+    }
+
+    .teachers-love-box {
+      text-align: center;
+      h3 {
```

Comment:
generally, I prefer not to put styles on HTML tags directly since if we add a new h3 element in this div, it would pick up the same styles. But this is not blocking the merge of this code.

Reply:
@mrfinch ok, I'll do so next time.